likeminds-feed-android-core/src/main/java/com/likeminds/feed/android/core/delete/view/LMFeedReasonChooseBottomSheetFragment.kt

```diff
@@ -0,0 +1,87 @@
+package com.likeminds.feed.android.core.delete.view
+
+import android.graphics.Color
+import android.os.Bundle
+import android.view.*
+import androidx.fragment.app.FragmentManager
+import androidx.fragment.app.viewModels
+import com.google.android.material.bottomsheet.BottomSheetDialogFragment
+import com.likeminds.feed.android.core.databinding.LmFeedReasonChooseBottomSheetFragmentBinding
+import com.likeminds.feed.android.core.delete.adapter.LMFeedReasonChooseAdapterListener
+import com.likeminds.feed.android.core.delete.model.LMFeedReasonChooseViewData
+import com.likeminds.feed.android.core.delete.viewmodel.LMFeedReasonChooseViewModel
+import com.likeminds.feed.android.core.utils.LMFeedViewUtils
+
+class LMFeedReasonChooseBottomSheetFragment : BottomSheetDialogFragment(),
+    LMFeedReasonChooseAdapterListener {
+
+    companion object {
+        private const val TAG = "LMFeedReasonChooseDialog"
+
+        @JvmStatic
+        fun newInstance(fragmentManager: FragmentManager) =
+            LMFeedReasonChooseBottomSheetFragment().show(fragmentManager, TAG)
+    }
+
+    private lateinit var binding: LmFeedReasonChooseBottomSheetFragmentBinding
+
+    private var reasonChooseDialogListener: LMFeedReasonChooseBottomSheetListener? = null
+
+    private val reasonChooseViewModel: LMFeedReasonChooseViewModel by viewModels()
+
+    override fun onCreateView(
+        inflater: LayoutInflater,
+        container: ViewGroup?,
+        savedInstanceState: Bundle?
+    ): View {
+        binding = LmFeedReasonChooseBottomSheetFragmentBinding.inflate(layoutInflater)
+
+        return binding.root
+    }
+
```

Comment:
no customization given?

Reply:
the customization of this fragment is inside the view data binder so there is no customization to give here